src/rtabmap_manager/params/rtabmap.yaml

```diff
@@ -0,0 +1,71 @@
+# RTAB-Map configuration
+
+subscribe_depth:        true
+subscribe_rgbd:         false
+subscribe_stereo:       false
+subscribe_stereo:       false
+subscribe_scan:         false
+subscribe_scan_cloud:   false
+subscribe_user_data:    false
+subscribe_odom_info:    false
+
+database_path:      "~/.ros/rtabmap.db"
+config_path:        "~/.ros/rtabmap.cfg"
+
+frame_id:       "base_link"
+map_frame_id:   "map"
+odom_frame_id:  "odom"                   # odometry from odom msg to have covariance - Remapped by launch file
+odom_tf_angular_variance:  0.001     # If TF is used to get odometry, this is the default angular variance
+odom_tf_linear_variance:   0.001     # If TF is used to get odometry, this is the default linear variance
+tf_delay:       0.02
+publish_tf:     false                # Set to false if fusing different poses (map->odom) with UKF
```

Comment:
I am pretty sure we want to set this to True since rtabmap is the one that is creating the map, thus we want it to also be the one publishing the map->odom transform.

The problem with that is, in its default configuration, the zed-ros-wrapper publishes this tf as well (see [`pos_tracking/publish_map_tf` here](http://wiki.ros.org/zed-ros-wrapper#Position_Parameters)) and we only want one publisher for this (rtabmap). 

So I would think that this would be an arg you can pass into `zed_camera.launch.xml` you call in line 16 of zed_host.launch. However, there isn't. When looking at the launch file provided by zed, they have [a launch file with no tfs](https://github.com/stereolabs/zed-ros-wrapper/blob/master/zed_wrapper/launch/include/zed_no_tf.launch.xml). BUT, I am pretty sure we want `pos_tracking/publish_tf` to be true and in that launch, it sets them both to false. 

So the way this is looking, we may need to use a modified version of `zed_camera.launch.xml` that has `pos_tracking/publish_tf=true` and `pos_tracking/publish_map_tf=false`.

If any of this is confusing, please feel free to ask.

Reply:
Good catch on this one. Added the new `zed_host.launch` in b97e7d612808792ffa372fa59ae09fe2268020b7. I compared the tf tree from before and after this change and everything seems to be in order! The only big difference is that RViz can no longer display the ZED data correctly until RTAB-Map is running alongside it, but I don't think its a real issue.

Reply:
Okay, nice job! I like the route of pulling in things we needed from `zed_camera.launch.xml` and just launching the zed_wrapper_node directly.

Two things:
1. Could you add the tf tree to this PR (in this comment thread or in the description)?
2. Can you please create a new bag file and upload it to your branch for #24?